pokemongo_bot/__init__.py

```diff
@@ -34,10 +34,15 @@ def start(self):
 
     def take_step(self):
         location = self.navigator.take_step()
-        cells = self.find_close_cells(*location[0:2])
+        self.work_cells(1, self.position[0:2])
 
-        for cell in cells:
-            self.work_on_cell(cell, location)
+  def work_cells(self, work_on_forts=1, location=None): 
```

Comment:
Please, name `work_cells` better. I deserves a cleaner naming, because `work_on_cell` already exists and this coexistence is not intuitive. 

Also you should not defined `work_on_forts` default value with 1 if it's boolean. Use `True` or `False`.